ehsm_kms_service/function.js

```diff
@@ -264,8 +264,10 @@ const enroll_user_info = (action, DB, res, req) => {
   if (napi_res) {
     const { appid, apikey } = napi_res.result
     let cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
-    if (cmk_res) {
+    let sm_default_cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
+    if (cmk_res && sm_default_cmk_res) {
       const { cmk } = cmk_res.result
+      const sm_default_cmk = sm_default_cmk_res.result.cmk

```

Comment:
add some comments here

Reply:
will be add: // create a default secret manager CMK for current appid.